packages/perseus-core/package.json

```diff
@@ -27,7 +27,7 @@
     "dependencies": {
         "@khanacademy/kas": "workspace:*",
         "@khanacademy/perseus-utils": "workspace:*",
-        "@khanacademy/simple-markdown": "workspace:*",
```

Comment:
I switched from simple to pure-markdown because pure-markdown is where we have Perseus-specific rules (such as `math` and `widgets`).  These aren't strictly necessary for what we're doing _now_ but I thought it was good to use the right set of parsing rules now so we don't create a hard-to-figure-out problem in the future.